### DISCUSSION TITLE
Not merging responsive modifiers

### DISCUSSION THREAD

**Discussion by abdessamadely** [dcastil/tailwind-merge#98] [Help] [answered]
# Sorry, I was giving it the wrong value

Please just close this one 🙏🏻

```
twMerge(
          'lg:max-w-5xl',
          'lg:max-w-3xl'
)
```
This one doesn't merge

version: `"tailwind-merge": "^1.2.1"`

**Comment by dcastil** [comment, marked as answer]
All good! 😊 Just not sure how to close a discussion. 🤔
